native/src/main/java/io/ballerina/persist/objects/Entity.java

```diff
@@ -62,6 +62,9 @@ public String[] getKeys() {
     }
 
     public String getTableName() {
+        if (this.tableName == null) {
+            this.tableName = entityName.toLowerCase();
```

Comment:
why are we changing the name to lowercase?

Reply:
Updated in [4526b78](https://github.com/ballerina-platform/persist-tools/pull/31/commits/4526b780825fc77aaee4b9eacf8276c9e0d9115e)